flake.nix

```
@@ -7,6 +7,16 @@
   };
 
   outputs = { self, nixpkgs, flake-utils }:
+    {
+      overlays.default = final: prev: {
+        prybar = {
```

```suggestion
        prybar = prev.lib.recurseIntoAttrs {
```
If we're going to have nesting, we probably want to make it so various nix tools can look for derivations in the attrset.

https://github.com/NixOS/nixpkgs/blob/0b34cc803f6b17282d728eea059fbe2a10a8f8fa/lib/attrsets.nix#L976